update_compilers/install_compilers.sh

```diff
@@ -147,38 +244,6 @@ install_rust 1.3.0
 install_rust 1.4.0
 
 
-#########################
```

Comment:
Can you put the go installation back please...I suspect you removed it during testing (quite reasonably...)

Reply:
Better yet! After putting poor go back in its place

> I thought about pulling out the Rust updating code into its own shell script. I was too lazy to do that, but I could be persuaded to not be lazy :)

We should move each language to its own file (Maybe have a "install_all_languages.sh" script too?) eventually so this does not happen again